content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -28,7 +28,7 @@ The following methods exist for installing kubectl on Linux:
 1. Download the latest release with the command:
 
    ```bash
-   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/$([ $(uname -m) = "aarch64" ] && echo "arm64" || echo "amd64" )/kubectl"
```

Comment:
This only handles two CPU architectures. I'm concerned that we shouldn't favor particular vendors (in this case: AMD, also Intel who have a clone of that ISA, and ARM). https://www.downloadkubernetes.com/ lists several other architectures and this command will download the wrong artifact there.

It's wrong less often, but the approach isn't the right one to take.

Reply:
How about adding a second sentence to the note below that reads something like: "Besides arm64 and amd64, versions include 386, arm, ppc64le, and s390x." This mentions all the versions, while keeping the command simple and limited to the most popular.

Reply:
That'd work. We should also list arm64 and amd64 as prerequisites, with a qualifying statement that there is a comment (that we add here) about using the alternative command and (eg) setting `$ARCHITECTURE` to the right value first.

Reply:
I would like to say if we can move a small step forward, let's do it. The current version is obviously worse than the pull request. Why stop it?  The amd64 and arm64 are the most popular archs. It may help a lot for people who move to arm64.